src/ring_theory/valuation/basic.lean

```diff
@@ -293,6 +293,13 @@ begin
   simpa only [v.map_one, v.map_neg] using v.map_add_eq_of_lt_left h
 end
 
+lemma one_lt_val_iff
+  {K : Type*} [division_ring K] (v : valuation K Γ₀) {x : K} (h : x ≠ 0) :
```

Comment:
This belongs in a new `division_ring` section, rather than in the `group` section.

Does it generalize to `group_with_zero`?